tool/travis.sh

```diff
@@ -16,6 +16,9 @@ pub global run tuneup check --ignore-infos
 # Run the tests.
 dart test/all.dart
 
+# Verify that the generated grind script analyzes well.
+dart tool/grind.dart analyze-init
```

Comment:
Eventually would be nice to do `pub run grinder:grind analyze-init` or even just `grind`, see #118.